### PR TITLE
[env] Introduce jni_helper for env_flink

### PR DESCRIPTION
**Pull request by masteryhx**
## What is the purpose of the change

- Introduce jni_helper to cache frequently-used jclasses

## Brief change log

  - Introduce jni_helper, define cached classes and init them

## Verifying this change

This is a tool class improving the jni performance which relys on the Flink environment. So it will be tested when env_flink is ready.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -728,6 +728,7 @@ set(SOURCES
         env/fs_remap.cc
         env/mock_env.cc
         env/unique_id_gen.cc
+        env/flink/jni_helper.cc
```
Comment (ljz2051) [inline CMakeLists.txt]: Should "env/flink/env_flink.cc" be placed near this position as well?

File: env/flink/jni_helper.cc [hunk 2]
```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "jni_helper.h"
+
+namespace ROCKSDB_NAMESPACE {
```
Comment (ljz2051) [inline env/flink/jni_helper.cc:8]: How about changing the namespace from ROCKSDB_NAMESPACE to FORST_NAMESPACE (or FLINK_NAMESPACE) ?

**Review by ljz2051 — COMMENTED** [review]
Thanks for the PR! I left some comments. PTAL.

File: CMakeLists.txt (hunk 1 above)
Comment (masteryhx) [reply CMakeLists.txt]: Thanks for the reminder. 
Moved.

File: env/flink/jni_helper.cc (hunk 2 above)
Comment (masteryhx) [reply env/flink/jni_helper.cc:8]: As discussed offline, we will handle this together later.

**Review by ljz2051 — APPROVED** [review]

**Pull request merged**